website/docs/d/cosmosdb_restorable_database_account.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+subcategory: "CosmosDB (DocumentDB)"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_cosmosdb_restorable_database_account"
```

Comment:
as this returns multiple i think this should be
```suggestion
page_title: "Azure Resource Manager: azurerm_cosmosdb_restorable_database_accounts"
```

as this returns restorable account ids

Reply:
Updated